nbs/models.dlinear.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #126.            seasonal_part = self.linear_trend(seasonal_init)
</pre><p>This should be linear_season</p><pre class="ql-syntax" spellcheck="false">seasonal_part = self.linear_season(seasonal_init)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>